web/carousel/src/components/PasswordForm/PasswordForm.module.css

```diff
@@ -1,5 +1,15 @@
-.ErrorMessage {
-    color: gray;
-    font-size: small;
-    width: 500px;
-}
\ No newline at end of file
+.Span {
+    display: flex;
+    height: 13px;
+    margin-left: 10px;
+}
+
+.PasswordCheck {
+    height: 8px;
+    font-size: 6px;
+    text-align: left;
+    align-items: center;
+    margin-top: -4px;
```

Comment:
We can remove this negative margin-top so that <p> element and the icons sit in the same horizontal line

Reply:
Make sense!